tests/std/tests/P0608R3_sane_variant_converting_constructor/test.cpp

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Also tests for P1957R2: Converting from T* to bool should be considered narrowing
+
+#include <assert.h>
+#include <bitset>
+#include <functional>
+#include <optional>
+#include <string>
+#include <type_traits>
+#include <variant>
+#include <vector>
+
+using namespace std;
+
+struct double_double {
+    double_double(double x) : x_(x) {}
+
+    double x_;
+};
+struct convertible_bool {
+    convertible_bool(bool x) : x_(x) {}
+    ~convertible_bool() = default;

```

Comment:
No change requested: Note that defaulting the destructor is unnecessary; it will be generated even if a constructor like `convertible_bool(bool x)` is declared. (Whereas the default constructor will "vanish" as soon as any other constructors are declared.)